package/MDAnalysis/topology/MOL2Parser.py

```diff
@@ -145,10 +157,26 @@ def parse(self, **kwargs):
             resids.append(resid)
             resnames.append(resname)
             charges.append(charge)
+            elements.append(atom_type.split(".")[0])
 
         n_atoms = len(ids)
 
-        masses = guessers.guess_masses(types)
+        validated_elements = []
+        invalid_elements = set()
+        for elem in elements:
+            if elem in SYMB2Z:
```

Comment:
I _think_ for the sake of being safe, it might make sense to just read elements as is and then compare them against a list of SYBYL atom types. That way if someone uses a weird MOL2 scheme that isn't an SYBYL but has `.` in the atom types, then we can make sure we treat them as unknowns.

Reply:
Good point. I added a `SYBYL2SYMB` dictionary for type to element conversion. I had to change some of the test `.mol2` files in order to conform to the standard (`S.O` and `S.O2` instead of `S.o` and `S.o2`). Should we support both cases?